pet-kata-solutions/src/test/java/org/eclipse/collections/petkata/Exercise5Test.java

```diff
@@ -70,6 +72,25 @@ public void addPetAgesToExistingSet()
         Assertions.assertEquals(expected, petAges);
     }
 
+    @Test
+    @Tag("SOLUTION")
+    @DisplayName("findOwnerWithMoreThanOnePetOfTheSameType - 🐹 🐹")
+    public void findOwnerWithMoreThanOnePetOfTheSameType()
```

Comment:
The formatting in this method does not match the project conventions. Please fix.